tests/pytest_runner.py

```diff
@@ -1,17 +1,29 @@
+import glob
 import os
 
 import pytest
 
-from yt.config import ytcfg
-
 if __name__ == "__main__":
     os.environ["OMP_NUM_THREADS"] = "1"
     pytest_args = [
-        f"--local-dir={os.path.join(ytcfg.get('yt', 'test_data_dir'), 'answers')}",
-        "-c=pytest_answer.ini",
-        "--junitxml=unittests.xml",
-        "--answer-big-data",
-        f"-n {int(os.environ.get('NUM_WORKERS', 6))}",
+        "-s",
+        "-v",
+        "-rsfE",
+        "--with-answer-testing",
+        "-m answer_test",
+        f"-n {int(os.environ.get('NUM_WORKERS', 1))}",
         "--dist=loadscope",
     ]
-    pytest.main(pytest_args)
+    pytest.main(pytest_args + ["--local-dir=answer-store", "--junitxml=answers.xml"])
+
+    if (files := glob.glob("generate_test*.txt")) :
+        tests = set()
+        for fname in files:
+            with open(fname) as fp:
+                tests |= set(fp.read().splitlines())
```

Comment:
I'm not 100% sure of the top of my head but I think `|=` is only supported for sets since Python 3.9 ? I'll check

Reply:
nope, it was for dictionaries only. Sets have been supporting this for much longer. Disregard !

Reply:
Judging by a green tick next to a test called yt_**py38**_answertests I'd say it's safe to bet it's supported by py3.8